services/requester/cadence/estimate_gas.cdc

```diff
@@ -0,0 +1,11 @@
+import EVM
+
+access(all)
+fun main(encodedTx: [UInt8]): [UInt64; 2] {
+    let coa <- EVM.createCadenceOwnedAccount()
```

Comment:
Better to use evm account resource:
```
    let account = getAuthAccount<auth(Storage) &Account>(Address(0xCOA))

    let coa = account.storage.borrow<auth(EVM.Call) &EVM.CadenceOwnedAccount>(from: /storage/evm)
        ?? panic("Could not borrow reference to the COA!")
```

Reply:
Good catch :+1: Fixed in https://github.com/onflow/flow-evm-gateway/pull/81/commits/eeea98e08eb9ddf17724ce2913dbe8b958a57c9a

I didn't use any entitlements, because `EVM.run` does not have any. It is simply defined as:
 ```cadence
 access(all) fun run(
 ```